keyboards/willow/willow64/readme.md

```diff
@@ -0,0 +1,18 @@
+# Willow64
+
+[![willow64](https://raw.githubusercontent.com/hanachi-ap/willow64-doc/main/img/img00.jpg)](https://github.com/hanachi-ap/willow64-doc/blob/main/img/img00.jpg)
+
+This is a 64 keys custom keyboard with a willow layout.
+
+- Keyboard Maintainer: [hanachi](https://github.com/hanachi-ap/) [@haswellep](https://twitter.com/haswellep)  
+- Hardware Supported:  Willow64 PCB, Pro Micro  
+- Hardware Availability: https://hanachiya.booth.pm/ 
+
+Make example for this keyboard (after setting up your build environment):
+
+    make willow/willow64:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you please add instructions above this line on how to reset the keyboard into bootloader mode?

Reply:
Thanks for the review.
Is this what your suggestion is?
`To enter bootloader mode, press the reset button on the keyboard.`

Reply:
Yes.

Reply:
Thank you. I fixed it.